README.md

```diff
@@ -20,7 +31,7 @@ to read secrets, which can both be used side-by-side:
 
 * Configure environment variables such as `VAULT_SECRET_PATH` for the extension
   to read a secret and write it to disk.
-* Make unauthenticated requests to the extension's local server at
+* **Recommended**: Make unauthenticated requests to the extension's local server at
```

Comment:
If we do recommend this, let's list it first

Reply:
We may also want to mention that with this method you're getting a response just like you would if you were talking to Vault directly, where as the `VAULT_SECRET_PATH` approach dumps values to files. With the Server method they're getting `api.Secret` values in an HTTP response

Reply:
Good points, we actually write an `api.Secret` out to disk for the pre-configured approach, so that data returned is very similar, just consumed by the lambda function differently.